gamemodes/terrortown/gamemode/shared/hud_elements/tttinfopanel/pure_skin_playerinfo.lua

```diff
@@ -62,6 +62,16 @@ if CLIENT then
                 slf:SaveData()
             end,
         }
+		
+		savingKeys.showTeamName = {
+            typ = "bool",
+            desc = "label_hud_show_team_name",
+            default = true,
```

Comment:
The default should be false. I'm fine with this being an option, but the team shouldn't be shown there by default